blessedDeps.gradle

```diff
@@ -51,7 +51,7 @@ rootProject.ext.PARIS_VERSION = "1.7.3"
 rootProject.ext.ROBOLECTRIC_VERSION = "4.5.1"
 rootProject.ext.SQUARE_JAVAPOET_VERSION = "1.13.0"
 rootProject.ext.SQUARE_KOTLINPOET_VERSION = "1.8.0"
-rootProject.ext.COMPOSE_VERSION = "1.0.0-beta06"
+rootProject.ext.COMPOSE_VERSION = "1.0.0-beta07"
```

Comment:
Using beta08 because I'm not sure if Epoxy can be upgraded to 1.5.1 yet. If its okay to do that, I'll also bump this up to the stable compose version -  `1.0.0 `